src/components/Utilities/GeneralUtil.js

```diff
@@ -24,7 +24,7 @@ const getNumDenValue = (memberValue) => {
 }
 
 const getMeasureCompliance = (memberResult) => {
-  const memberDetails = memberResult.result;
+  const memberDetails = memberResult[memberResult.memberId];
```

Comment:
You need to update your local HeRA to the latest develop branch, recreate the test data in Mongo, and test this again. You might find some issues, like here.